packages/@uppy/locales/src/ar_SA.ts

```diff
@@ -1,5 +1,8 @@
-const ar_SA = {
-  pluralize (n) {
+import type { Locale } from '@uppy/utils/lib/Translator'
```

Comment:
That forcees TS user to have `@uppy/utils`, should we make it a peerDependency?

Reply:
Why not a depedency?

Reply:
If our public facing types are from a package that that sounds exactly like a dependency to me?

Reply:
It still saddens me to see a package go from 0 dependency to 1. But fair enough